app/controllers/admin/base_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Admin::BaseController < ApplicationController
```

Comment:
I think we can conform to the `module Admin`/ `class BaseController` that hound is complaining about here.